.eleventy.js

```diff
@@ -157,6 +160,17 @@ module.exports = function(config) {
 			})
 	})
 
+  config.addCollection('definedWordsChronological', (collection) => {
+    return collection
+      .getFilteredByGlob('./11ty/definitions/*.md')
+      .filter((word) => word.data.defined)
+      .sort((a, b) => {
+        if (a.date > b.date) return 1
+        if (a.date < b.date) return -1
+        return 0;
```

Comment:
The sort should be the other way around so that we have the newest item first.

Witht this the updated timestamp looks like `<updated>2020-05-17T15:04:44.465+02:00</updated>`. But the timestamp of the newest item is `<updated>2020-06-10T18:02:15.575+02:00</updated>`.

Reply:
Makes sense! Pushing up the change now!